codalab/model/bundle_model.py

```diff
@@ -1726,6 +1726,41 @@ def get_user(self, user_id=None, username=None, check_active=True):
         if result:
             return result[0]
         return None
+    
+    def search_users(self, user_id, keywords):
+        clauses = []
+        limit = 5
+        for keyword in keywords:
+            if '.limit' not in keyword:
```

Comment:
I'm not sure the best way to filter out filters (e.g. `.limit` or `.sort=reverse`). Thoughts?

Reply:
A hack is to just ignore anything that starts with a period.  Definitely should make this more generic.  If we're not going to respect `limit`, then we should document that.